litmus-portal/frontend/src/components/ProgressBar/AnalyticsLinearProgressBar/index.tsx

```diff
@@ -13,17 +13,34 @@ const AnalyticsLinearProgressBar: React.FC<LinearProgressBarProps> = ({
   maxValue,
   isInTable,
 }) => {
-  const width: number = 2;
+  const width: number = 15; //default width
   const resultValue = ((value as number) / (maxValue as number)) * 100;
 
-  const { palette } = useTheme();
+  const barStyle = makeStyles(() => ({
+    root: {
+      borderRadius: 5,
+      height: width,
+    },
+    barColorPrimary: {
+      backgroundColor:
+        resultValue > 60 ? '#109B67' : resultValue > 30 ? '#F6B92B' : '#CA2C2C',
+    },
+  }));
```

Comment:
please check other components and create a similar corresponding styles.ts file instead of keeping css in the same index file. Also use `#F6B92B` and `#CA2C2C` from `palette`, other changes look good.